descartes_core/package.xml

```diff
@@ -17,11 +17,13 @@
   <build_depend>console_bridge</build_depend>
   <build_depend>moveit_core</build_depend>
   <build_depend>roscpp</build_depend>
+  <build_depend>cmake_modules</build_depend>
 
   <run_depend>boost</run_depend>
   <run_depend>console_bridge</run_depend>
   <run_depend>moveit_core</run_depend>
   <run_depend>roscpp</run_depend>
+  <run_depend>cmake_modules</run_depend>
```

Comment:
I don't believe a `run_depend` is needed with `cmake_modules`. It's a build-only thing.